omnigibson/examples/robots/advanced/ik_example.py

```diff
@@ -38,7 +38,12 @@ def main(random_selection=False, headless=False, short_exec=False):
         programmatic_pos = True
 
     # Import scene and robot (Fetch)
+    scene_cfg = {"type": "Scene"}
+    cfg = dict(scene=scene_cfg)
+    env = og.Environment(configs=cfg)
     scene = Scene()
+    if og.sim.is_playing():
+        og.sim.stop()
```

Comment:
All this is quite weird. The env already has a scene so why import a new one, also why stop it and import a robot afterwards? Lets put the scene and the robot in the config, and don't make any additional stop/import/etc. calls.